src/common.rs

```diff
@@ -367,6 +367,18 @@ pub struct LoadAvg {
 /// ```
 #[derive(PartialEq, Eq, PartialOrd, Ord, Debug)]
 pub struct User {
+    #[cfg(not(windows))]
+    /// User ID
+    pub uid: libc::uid_t,
```

Comment:
Just realized: I tend to use getters instead of allowing access to fields directly. Could you do that please?

To make it work, you'll need to create a newtype (or an alias, as you prefer) so that the `UserExt` trait will be able to return it.

Reply:
Added `Uid` and `Gid` newtypes to handle this.